### PR TITLE
chore: update CLI targeting and add requirements to readme

### LINKED ISSUE #1525: What versions of .NET does this work with?
I got asked if this works with 3.1 or 5 or both. It doesn't say anywhere in the docs.

### PR DESCRIPTION
**Pull request by avodovnik**
Fixes #1525

### CODE REVIEW

**Comment by avodovnik** [comment]
Note, I've targeted `netcoreapp3.0` here, because it's the minimum where `range` is supported. I don't think we need to go any lower unless we get actual requests for it.

**Review by kblok — APPROVED** [review]

File: src/Playwright.CLI/Playwright.CLI.csproj [hunk 1]
```diff
@@ -20,8 +20,8 @@
     <IsPackable>true</IsPackable>
   </PropertyGroup>
   <PropertyGroup>
-    <AssemblyVersion>1.0.0</AssemblyVersion>
-    <PackageVersion>$(AssemblyVersion)-alpha-5</PackageVersion>
+    <AssemblyVersion>1.0.1</AssemblyVersion>
```
Comment (pavelfeldman) [inline src/Playwright.CLI/Playwright.CLI.csproj:23]: We should not land stable package versions in trunk, so this should become -alpha-6. You won't even be able to cherry-pick it like this, and for a good reason.

**Review by pavelfeldman — CHANGES_REQUESTED** [review]
If that is the only reason, I'd replace this one line with a 3-line for loop. I had done it locally.

File: src/Playwright.CLI/Playwright.CLI.csproj (hunk 1 above)
Comment (avodovnik) [reply src/Playwright.CLI/Playwright.CLI.csproj:23]: Changed it to `-next-1`.

**Pull request merged**